### PR TITLE
HTML: Tests for button layout (WIP)

### PR DESCRIPTION
**Pull request by zcorpan**
Follows https://github.com/whatwg/html/pull/4143.

(More tests will be added to this PR.)

### CODE REVIEW

**Comment by zcorpan** [comment]
I'm not sure what happened here. This and #14824 both result in 404 for me when trying to view them. (They loaded when I submitted the PR but then I couldn't add labels, and reloading the page gave 404.)

File: html/rendering/non-replaced-elements/form-controls/button-style.html [hunk 1]
```diff
@@ -0,0 +1,20 @@
+<!doctype html>
+<title>default style on buttons</title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
```
Comment (javifernandez) [inline html/rendering/non-replaced-elements/form-controls/button-style.html]: You should add a comment, using the 'meta' tag and  'assert' id,  to explain briefly what the test tries to verify.

File: html/rendering/widgets/button-layout/computed-style.html [hunk 2]
```diff
@@ -0,0 +1,31 @@
+<!doctype html>
+<title>computed style on buttons</title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
```
Comment (javifernandez) [inline html/rendering/widgets/button-layout/computed-style.html]: Ditto

**Comment by jgraham** [comment]
Adding `<meta name=assert>` is explictly not required in web-platform-tests (although authors are of course free to do that if they like). Some specs may try to impose stronger requirements here, but for HTML there isn't any such rule.

File: html/rendering/non-replaced-elements/form-controls/button-style.html (hunk 1 above)
Comment (zcorpan) [reply html/rendering/non-replaced-elements/form-controls/button-style.html]: Isn't it clear from the title and the test names?

**Comment by zcorpan** [comment]
Please continue discussion / review in https://github.com/web-platform-tests/wpt/pull/14824

**Pull request closed without merging**